app/resources/static/fixed/forms.js

```diff
@@ -361,6 +361,22 @@ function validate_fields() {
     }
   }
 
+  // Validate age format
+  var age_field = $('age');
+  var age_value = age_field.value;
+
+  if (age_value != "") {
+    var curated_age_value = age_value.replace(/\s/g,'');
+    var valid_age_re = /^(\d+-?)+\d+$/;
+
+    var isAgeFormatValid = valid_age_re.test(curated_age_value);
+
+    if (!isAgeFormatValid) {
+      alert("The age entered is not in a valid format.");
```

Comment:
We'd like to be consistent about how we display errors, so could you please handle this like the others. The improper email address one is probably a good example you could follow: it's displayed with code a little bit above in this file, and the HTML is in `app/resources/errors.html.template`.